.github/workflows/Coverage.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverage
+
+on: [pull_request]
+
+jobs:
+  coverage:
+    runs-on: macos-latest
```

Comment:
This increases the cost by 10x compared to Linux. Are we sure we want to do this? Maybe just for every merge instead of every PR update?

Reply:
Nevermind. https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions 

> GitHub Actions usage is free for standard GitHub-hosted runners in public repositories, and for self-hosted runners